cellpose/gui/menus.py

```diff
@@ -75,6 +75,17 @@ def mainmenu(parent):
     file_menu.addAction(parent.saveSettings)
     parent.saveSettings.setEnabled(True)
 
+    """
+    This adds a new menu item for saving features as a .csv file. 
+    The user can activate this function to export specific data directly from the GUI.
+    The function `_save_features_as_csv` from the `io` module is called when the user clicks on the menu item.
+    """
+    parent.saveFeaturesCsv = QAction("Save Features as .&csv", parent)
+    parent.saveFeaturesCsv.setShortcut("Ctrl+C")
```

Comment:
Shortcut not ideal because it is usually reserved for copying (though I don't have a better idea at the moment either)